cst438-project-airline/src/main/java/cst438/controller/AirlineController.java

```diff
@@ -64,31 +64,33 @@ public String pickSeats(@RequestParam("flightId") int flightId,
     model.addAttribute("firstClassSeats", firstClassSeats);
     model.addAttribute("coachSeats", coachSeats);
 
-
     return "pick_seats";
 
   }
 
   @PostMapping("/searchFlights/passengers")
   public String passengers(@RequestParam("flightId") String flightId,
-      @RequestParam(value = "numberOfPassengers") int numberOfPassengers, Model model) {
+      @RequestParam("numberOfPassengers") int numberOfPassengers,
+      @RequestParam("selectedSeats[]") ArrayList<String> selectedSeatIds, Model model) {
```

Comment:
Just out of curiosity, is the [] doing something special in Thymeleaf or was it just your naming choice?